sources/packages/backend/apps/api/src/route-controllers/overaward/_tests_/e2e/overaward.students.controller.getOverawardsByStudent.e2e-spec.ts

```diff
@@ -55,40 +80,49 @@ describe("OverawardStudentsController(e2e)-getOverawardsByStudent", () => {
     application.currentAssessment = studentAssessment;
     await applicationRepo.save(application);
     // Create an overaward.
-    const reassessmentOveraward = createFakeDisbursementOveraward({ student });
+    let reassessmentOveraward = createFakeDisbursementOveraward({ student });
```

Comment:
I would suggest creating a second variable, at least from what I have seen for the tests.